File: BE/HouseKepperConnect/BusinessObject/DTO/HouseKeeperDisplayDTO.cs
using System.ComponentModel.DataAnnotations;

namespace BusinessObject.DTO
{
    public class HouseKeeperDisplayDTO
    {
        public int HousekeeperID { get; set; }
        public int AccountID { get; set; }
        public string Name { get; set; }
        public int Gender { get; set; }

        [EmailAddress]
        public string Email { get; set; }

        [Phone]
        public string? Phone { get; set; }

        public int? WorkType { get; set; }

        public string LocalProfilePicture { get; set; }
        public string? GoogleProfilePicture { get; set; }

        [RegularExpression("^[0-9]*$", ErrorMessage = "Bank number must be numeric.")]
        public string? BankAccountNumber { get; set; }
        public string? BankAccountName { get; set; }

        public string? Introduction { get; set; }

        public int? VerifyID { get; set; }
        public string Address { get; set; }

        public string FrontPhoto { get; set; }

        public string BackPhoto { get; set; }

        public string FacePhoto { get; set; }
    }
}

File: Mobile/app/src/main/java/com/example/housekeeperapplication/Model/DTOs/HousekeeperDisplayDTO.java
package com.example.housekeeperapplication.Model.DTOs;

public class HousekeeperDisplayDTO {
    private int housekeeperID;
    private int accountID;
    private String name;
    private String email;
    private String phone;
    private String frontPhoto;
    private String backPhoto;
    private String facePhoto;

    public String getFrontPhoto() {
        return frontPhoto;
    }

    public void setFrontPhoto(String frontPhoto) {
        this.frontPhoto = frontPhoto;
    }

    public String getFacePhoto() {
        return facePhoto;
    }

    public void setFacePhoto(String facePhoto) {
        this.facePhoto = facePhoto;
    }

    public String getBackPhoto() {
        return backPhoto;
    }

    public void setBackPhoto(String backPhoto) {
        this.backPhoto = backPhoto;
    }
}
